src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyAIService.js

```diff
@@ -115,16 +97,12 @@ class ThirdPartyAIService extends Component {
 
 const mapStateToProps = state => ({
   grpcResponse: state.serviceReducer.serviceMethodExecution.response,
```

Comment:
Do you need this?

Reply:
done https://github.com/singnet/refactored-dapp/pull/171/commits/2f115408b3dd5126d1e97a6967b001d41a2fcc0c